packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -105,6 +111,16 @@ function InterfaceSkeleton(
 							{ secondarySidebar }
 						</div>
 					) }
+					{ !! notices && (
+						<div
+							className="interface-interface-skeleton__notices"
+							role="region"
```

Comment:
I wonder if it should be labelled region, because right now, when you use "region navigations" (ctrl + @ on my Mac but I know it can be different in different keyboard layouts), you'll see an "empty region" stop. It's not the only one though, it seems there's another empty region somewhere. 

Maybe we can start without a region role label and tabIndex

Reply:
We can "promote" it to a full region once all notifications are rendered here.

Reply:
Should I remove the `aria-label` as well?

Reply:
yep it's useless if it's not a region and no tabIndex

Reply:
Updated. I think the new tab region also caused the e2e test failure. Hopefully, that's resolved now.